samples/common/template/consumer_settings.yaml

```diff
@@ -0,0 +1,14 @@
+#
+# Consumer Settings
+#
+
+# The IP address and port number that the provider listens on for provider requests.
+# Example: "0.0.0.0:80"
+provider_authority: <<value>>
```

Comment:
update comment and param to use "consumer"

Reply:
Fixed.